.github/workflows/ci.yml

```diff
@@ -43,6 +43,13 @@ jobs:
           matrix_total: ${{ matrix.total }}
           address: ${{ fromJSON(steps.tflocal.outputs.workspace-outputs-json).tfe_address }}
           token: ${{ fromJSON(steps.tflocal.outputs.workspace-outputs-json).tfe_token }}
+          admin_configuration_token: ${{ steps.tflocal.outputs.workspace-outputs-json.tfe_admin_token_by_role.configuration }}
+          admin_provision_licenses_token: ${{ steps.tflocal.outputs.workspace-outputs-json.tfe_admin_token_by_role.provision_licenses }}
+          admin_security_maintenance_token: ${{ steps.tflocal.outputs.workspace-outputs-json.tfe_admin_token_by_role.security_maintenance }}
+          admin_site_admin_token: ${{ steps.tflocal.outputs.workspace-outputs-json.tfe_admin_token_by_role.site_admin }}
+          admin_subscription_token: ${{steps.tflocal.outputs.workspace-outputs-json.tfe_admin_token_by_role.subscription }}
+          admin_support_token: ${{ steps.tflocal.outputs.workspace-outputs-json.tfe_admin_token_by_role.support }}
+          admin_version_maintenance_token: ${{ steps.tflocal.outputs.workspace-outputs-json.tfe_admin_token_by_role.version_maintenance }}
```

Comment:
The CI itself isn't giving you the output you need to verify this, but you can see in the action log on this PR that these values aren't present, even though they are present in the workspace outputs:

![Screenshot 2023-05-11 at 2 44 12 PM](https://github.com/hashicorp/go-tfe/assets/174332/af4f4160-e140-466c-9ede-d3f5bcc1b06e)

The problem, I think, is that `steps.tflocal.outputs.workspace-outputs-json` comes out as a string (all GHA I/O are strings) so you have to use expressions like `fromJSON(steps.tflocal.outputs.workspace-outputs-json).tfe_admin_token_by_role.configuration`

Reply:
Ah - I was wondering why I wasn't seeing the variables in the outputs. OK. Another update coming.